runtime/interpreter/value.go

```diff
@@ -6006,6 +6006,90 @@ func (AddressValue) SetMember(_ *Interpreter, _ LocationRange, _ string, _ Value
 	panic(errors.NewUnreachableError())
 }
 
+// AccountKeyValue
+
+type AccountKeyValue struct {
+	publicKey []byte
```

Comment:
This is missing the signature algorithm, hash algorithm, and weight fields. 

We can add them later, in that case remove the fields from now here:
https://github.com/onflow/cadence/pull/159/files#diff-53914ebd61e0e93500b2ecb75a987b65R4697-R4704